lib/Dialect/Torch/Transforms/RefineTypes.cpp

```diff
@@ -224,7 +224,7 @@ class TypeAnalyzer : public ForwardDataFlowAnalysis<ValueKnowledge> {
   visitOperation(Operation *op,
                  ArrayRef<LatticeElement<ValueKnowledge> *> operands) final {
     if (isa<TensorStaticInfoCastOp, CopyToValueTensorOp, CopyToNonValueTensorOp,
-            AtenTanhOp, AtenBatchNormOp, AtenReluOp, AtenGeluOp,
+            AtenTanhOp, AtenBatchNormOp, AtenReluOp, AtenLeakyReluOp, AtenGeluOp,
```

Comment:
The op involves both tensor and scalar so should use `visitBinaryTensorScalarOp` to take care of type promotion.